src_testbed/testbed.rs

```diff
@@ -464,6 +429,7 @@ impl Testbed {
             }
         }
 
+        //FIXME: move this to dedicated benchmarking code
```

Comment:
```suggestion
        // TODO: move this to dedicated benchmarking code
```
Yeah, this should at least be in a separate function, on a separate module. But this is not important for this PR.